dev-libs/cdk/cdk-5.0.20230201-r2.ebuild

```diff
@@ -15,7 +15,7 @@ SRC_URI+=" verify-sig? ( https://invisible-island.net/archives/${PN}/${MY_P}.tgz
 S="${WORKDIR}"/${MY_P}
 
 LICENSE="MIT"
-SLOT="0/6" # subslot = soname version
+SLOT="0/6.3.4" # subslot = soname version, check VERSION
```

Comment:
I would make the comment say that it can be changed to just 7 on the next SONAME change?

Reply:
Yeah, you're right. OK.